src/app.ts

```diff
@@ -26,6 +27,9 @@ export default class App {
     const accessTokenServiceRESTApi = await AccesstokenServiceManager.createRestAPIServer();
     this.app.use('/', accessTokenServiceRESTApi);
 
+    const taskServiceRESTApi = await TaskServiceManager.createRestAPIServer();
```

Comment:
Rebase to pull the latest code and move this to appropriate function so it can be mounted to `/api`